pkg/scheduler/api/resource_info.go

```diff
@@ -109,6 +109,25 @@ func (r *Resource) Sub(rr *Resource) *Resource {
 		r, rr))
 }
 
+//Computes the delta between a resource oject representing available
+//resources an operand representing resources being requested.  Any
+//field that is less than 0 after the operation represents an
+//insufficient resource.
+func (r *Resource) FitDelta(rr *Resource) *Resource {
```

Comment:
this should be a helper function in scheduler instead of a func of resources. `FitDelta` is more about scheduling part.

Reply:
makes sense

Reply:
on a second thought though, it is generic in the sense that it computes the delta or deficit of resource regardless of the context ... also it depends on a number of intimate details of resources (such as minMilliCPU, etc.) which would need to be exported if we put that in a different package ..
thoughts ?

Reply:
what's different with `Sub`?

Reply:
`Sub` does not allow for negative results (the left operand must be greater or equal to the right operand).